.github/workflows/cd.yml

```diff
@@ -0,0 +1,98 @@
+name: CD
+
+on:
+  push:
+    tags-ignore:
+      - "*.dev0"
+  pull_request:
+    branches:
+      - develop
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  Build:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        include:
+          - arch: x64
+            wheel_arch: amd64
```

Comment:
Is this being used? Remove if not

I was just thinking that the actual wheel platform is `win_amd64` and a better name would have been `py_platform` since it's platform tag generated by Python.